### DISCUSSION TITLE
添加博客看板

### DISCUSSION THREAD

**Discussion by imjason** [Qexo/Qexo#122] [Ideas]
就文章量，访客，action运行状况啥的，
纯属找事(逃)

**Comment by imjason** [comment]
不要暂不修改了，求求了

**Comment by am-abudu** [comment]
暂定为/pub/status/
与原/pub/last/融合 下一版本支持 仅支持文章量和博主最后登录时间

**Reply by am-abudu** [reply]
访问量不提供统计

**Reply by imjason** [reply]
action运行状况呢

**Reply by am-abudu** [reply]
> action运行状况呢

因为很多人用的不是action 这玩意不好适配
